Client/Telemetry/UnifiedTelemetry.swift

```diff
@@ -201,6 +201,8 @@ extension UnifiedTelemetry {
         case dismissedOnboardingEmailLogin = "dismissed-onboarding-email-login"
         case dismissedOnboardingSignUp = "dismissed-onboarding-sign-up"
         case privateBrowsingButton = "private-browsing-button"
+        case TodayWidgetNewSearch = "today-widget-new-search"
```

Comment:
Since we don't need to add events lets clean up this part of the code related to telemetry. 
We can revisit it later on.